SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -312,20 +294,18 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::FlowSurfaceMesh(const std::
     mass_filter->SetInputData(mesh);
     mass_filter->Update();
     double original_volume = mass_filter->GetVolume();
-//    std::cout << "Original Volume: " << original_volume << std::endl;
-
     // default parameters
     // double dt = 0.001;
     // double smooth_amount = 0.03;
-    // int max_iter = 300;
+    // int max_iter = 500;
     int iter = 0;
     double tolerance = 0.05;
     double q = 1.0;
 
     // create folder if not exist
-    char forwardFolder[MAX_FILE_NAME];
     const char *tempFolder = this->GetApplicationLogic()->GetTemporaryPath();
```

Comment:
Could you use?

const std::string tempFolder(this->GetApplicationLogic()->GetTemporaryPath());